src/test/java/org/apache/commons/codec/language/MatchRatingApproachEncoderTest.java

```diff
@@ -519,4 +520,34 @@ public final void testRemoveVowel_ALESSANDRA_Returns_ALSSNDR() {
 
     // ***** END REGION - TEST GET MRA COMPARISONS
 
+    // ***** BEGIN REGION - TEST INVALID STRING *****
+
+    @Test
+    public final void testPunctuationOnly() {
+        assertDoesNotThrow(() -> {
+            String expected = "";
```

Comment:
No need for local vars, block, or does not throw api in single assert method. I've already commented on this in a separate PR.

Reply:
Resolved.